lib/shopify-cli/tasks/select_org_and_shop.rb

```diff
@@ -0,0 +1,79 @@
+require 'shopify_cli'
+
+module ShopifyCli
+  module Tasks
+    class SelectOrgAndShop < ShopifyCli::Task
+      attr_reader :ctx
+
+      def call(ctx, organization_id: nil, shop_domain: nil)
+        @ctx = ctx
+        return response(organization_id.to_i, shop_domain) if !organization_id.nil? && !shop_domain.nil?
+        org = get_organization(organization_id)
+        shop_domain = get_shop_domain(org, shop_domain)
```

Comment:
I wonder if it makes sense to pass in `shop_domain` as a parameter here - if the method just returns it when set, should we just not call the method at all `if shop_domain.nil?`?